### PR TITLE
add gcloud action

### PR DESCRIPTION
**Pull request by andreavs**
## Describe your changes
Added a new action to setup the standard google sdk stuff. 

I also broke the existing convention by putting the action at the root level of the repo. I think this makes more sense since we need to include the full folder path of the action when invoking it, but I might have missed something.

## How did you test it
added a test workflow

### CODE REVIEW

File: .github/workflows/gcloud-sdk-test.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2   
+    - name: Install dependencies
+      uses: 'hagaltech/common-workflows/setup-gcloud-sdk@add-gcloud-action'
+      with:
+        gcp_credentials: ${{ secrets.GCP_SA_KEY }}
+      
+    - run: |
+        gcloud info
```
Comment (katyamust) [inline .github/workflows/gcloud-sdk-test.yml]: Should we expose path to our registry here as this is public repo? May be we can consider to it is SECRET?

File: setup-gcloud-sdk/action.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      with:
+        credentials_json: '${{ inputs.gcp_credentials }}'
+
+    - name: 'Set up Cloud SDK'
+      uses: 'google-github-actions/setup-gcloud@v0'
+
+    - run: |
+        gcloud config set artifacts/repository hagal
+        gcloud config set artifacts/location europe-west1
+        gcloud auth configure-docker europe-docker.pkg.dev
```
Comment (katyamust) [inline setup-gcloud-sdk/action.yml:25]: Same here, not sure how our artifacts locatio and repo data is sensitive.  Leaving it to your judgement.

**Review by katyamust — APPROVED** [review]

**Review by tordf — APPROVED** [review]
Looks great 👍.  have same question as Katya about paths, and as she said, I'm not sure how critical it it.

File: setup-gcloud-sdk/action.yml (hunk 2 above)
Comment (andreavs) [reply setup-gcloud-sdk/action.yml:25]: I think i will leave this as is - it does not reveal too much, only our name and that we are located in europe.

File: .github/workflows/gcloud-sdk-test.yml (hunk 1 above)
Comment (andreavs) [reply .github/workflows/gcloud-sdk-test.yml]: Made the registry name a secret :+1:

**Pull request merged**